torch/_inductor/async_compile.py

```diff
@@ -226,7 +227,10 @@ def triton(self, kernel_name: str, source_code: str, device_str: str = "cuda"):
                 ),
             )
         else:
-            kernel.precompile()
+            with dynamo_timed(
+                "triton_compile", log_pt2_compile_event=False, fwd_only=False
+            ):
+                kernel.precompile()
```

Comment:
Deeply suspicious about this, not only because there's a lot of these, but also because I don't know if this is blocking or not. Why not just do an event for the module load outside of this

Reply:
Looks like empirically it's ok

Reply:
Honestly, this event isn't that important to track, even just in tlparse, will revert it for now and keep the other part